stylesheets/javascript/checkout-js.js

```diff
@@ -0,0 +1,13 @@
+function submitClick() {
```

Comment:
You seem to be using Jquery, you are required to do this for a later assignment but not #8. I have not looked ahead if you already submitted that or not. Nice work, my only addition would be to e.prevent default to keep the page from refreshing when you use the form. We do not want that behavior for this assignment.